Source/AtomicEditor/Editors/JSResourceEditor.cpp

```diff
@@ -188,6 +191,16 @@ void JSResourceEditor::HandleWebMessage(StringHash eventType, VariantMap& eventD
                 file.Write((void*) code.CString(), code.Length());
                 file.Close();
             }
+            else if (message == EDITOR_GET_USER_PREFS)
+            {
+                ToolSystem* tsys = GetSubsystem<ToolSystem>();
+                Project* proj = tsys->GetProject();
+                FileSystem* fileSystem = GetSubsystem<FileSystem>();
+                if (fileSystem->FileExists(proj->GetUserPrefsFullPath()))
+                {
+                    webClient_->ExecuteJavaScript(ToString("HOST_loadPreferences(\"atomic://%s\");", proj->GetUserPrefsFullPath().CString()));
+                }
```

Comment:
The reason that this is still here is that there needs to be some kind of notification event to the typescript indicating that the page is fully loaded.  Currently, there is not so the typescript can't execute any javascript on editor creation because those methods don't exist yet.  Ideally this will get removed, I just need to think on how to notify the typescript when the page is done loading...possibly in the EDITOR_LOAD_COMPLETE block above.  Maybe do a sendEvent with EDITOR_LOAD_COMPLETE that can be hooked into.